daemon/src/collab_settlement/current.rs

```diff
@@ -0,0 +1,5 @@
+pub mod maker;
+pub mod protocol;
+pub mod taker;
+
+pub const PROTOCOL: &str = "/itchysats/collab-settlement/2.0.0";
```

Comment:
Since you introduce 0.2.0 we could go for a breaking change here and implement: https://github.com/itchysats/itchysats/issues/2924 :)

Reply:
Sure, we should prioritise that ticket before we release something with these changes.